aiohttp_pydantic/oas/view.py

```diff
@@ -132,6 +132,10 @@ def _add_http_method_to_oas(
             return_type
         )
 
+    customization = getattr(handler, '__modify_schema__', None)
```

Comment:
Other alternatives are of course possible but this has the most concrete context. On `PydanticView` a similar hook could post-process all methods. This hook could also be added as an annotation instead, however this Python decorator allows code reuse without requiring intermediate class declarations of sorts or otherwise interfering with the interpretation of annotations, which are already being used for the usual typing rules. This avoids conflicting with the prior annotations that may gain the ability to describe examples etc. via other means in the future. Recall that the primary usecase is a fallback.

Reply:
It's good for me